releases/release-1.31/release-notes/maps/pr-124598-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124598
+releasenote:
+  text: kubectl describe service and ingress will now use endpointslices instead of
```

```suggestion
  text: `kubectl describe service` and `kubectl describe ingress` will now use endpointslices instead of
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653331570